render/theinternet.go

```diff
@@ -2,11 +2,50 @@ package render
 
 import (
 	"net"
+	"regexp"
 
 	"github.com/weaveworks/scope/probe/host"
 	"github.com/weaveworks/scope/report"
 )
 
+var (
+	// ServiceNodeIDPrefix is how the ID all service pseudo nodes begin
+	ServiceNodeIDPrefix = "service-"
+
+	knownServicesMatchers = []*regexp.Regexp{
+		// See http://docs.aws.amazon.com/general/latest/gr/rande.html for fainer grained
+		// details
+		regexp.MustCompile(`^.+\.amazonaws\.com$`),
+		regexp.MustCompile(`^.+\.googleapis\.com$`),
+	}
+
+	knownServicesExcluder = []*regexp.Regexp{
+		// We exclude ec2 machines because they are too generic
+		// and having separate nodes for them makes visualizations worse
+		regexp.MustCompile(`^ec2.*\.amazonaws\.com$`),
+	}
+)
```

Comment:
The matchers/excludersd should really be obtained by the probes. For the moment I'll settle for a 'TODO' and associated issue.